frontend/src/components/authentication/history.js

```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from "history";
```

Comment:
Can you write a comment explaining what this is for?

Reply:
Added comments in commit #https://github.com/code-for-canada/project-thundercat/pull/312/commits/3176cff3c1af31353290a9afb7691050f2993c4f